katello_certs_tools/sslToolConfig.py

```diff
@@ -740,23 +728,22 @@ def save(self, d, caYN=0, verbosity=0):
               % (gen_req_distinguished_name(rdn), d['--purpose'],  gen_req_alt_names(d, rdn['CN']))
 
         try:
-            rotated = rotateFile(filepath=self.filename,verbosity=verbosity)
-            if verbosity>=0 and rotated:
-                print "Rotated: %s --> %s" % (os.path.basename(self.filename),
-                                              os.path.basename(rotated))
+            rotated = rotateFile(filepath=self.filename, verbosity=verbosity)
+            if verbosity >= 0 and rotated:
+                print("Rotated: %s --> %s" % (os.path.basename(self.filename),
+                                              os.path.basename(rotated)))
         except ValueError:
             pass
         fo = open(self.filename, 'w')
         fo.write(openssl_cnf)
         fo.close()
-        os.chmod(self.filename, 0600)
+        os.chmod(self.filename, 0o600)
         return openssl_cnf
 
 
 ##
-## generated RPM "configuration" dumping ground:
+# generated RPM "configuration" dumping ground:
 ##
-
 POST_UNINSTALL_SCRIPT = """\
 if [ \$1 = 0 ]; then
```

Comment:
my `flake8` complains `W605 invalid escape sequence '\$'`, maybe make this a raw string (`P_U_S = r"""…"""`)

Reply:
which is interesting, as you have disabled it later on

Reply:
I don't entirely follow what the suggested change is here

Reply:
ignore it for now, it's not important :)